pkg/functions/parse.go

```diff
@@ -322,3 +329,39 @@ func ParseFunctionCall(llmresult string, functionConfig FunctionsConfig) []FuncC
 
 	return results
 }
+
+func ParseFunctionCallArgs(functionArguments string, functionConfig FunctionsConfig) string {
+	if len(functionConfig.ArgumentRegex) > 0 {
```

Comment:
A small nit here - instead of branching I would propose something the lines of:

```go
if len(functionConfig.ArgumentRegex) == 0 {
  return functionArguments
}

// Here the logic to handle argument regexes
```
```

Reply:
> A small nit here - instead of branching I would propose something the lines of:
> 
> ```go
> if len(functionConfig.ArgumentRegex) == 0 {
>   return functionArguments
> }
> 
> // Here the logic to handle argument regexes
> ```

That's a good point!
I'll open another PR to change that, as well as for the templates in the gallery.